libs/designsystem/src/lib/components/card/card-header/card-header.component.ts

```diff
@@ -9,8 +9,15 @@ export class CardHeaderComponent implements OnInit {
   @Input() title: string;
   @Input() subtitle: string;
   @Input() isTitleBold: boolean;
+  @Input() flagged: 'success' | 'warning' | 'danger' | 'informational' = null;
 
   constructor() {}
 
   ngOnInit() {}
+
+  @HostBinding('class')
+  get _cssClass() {
+    // TODO: Should we use the 'flagged-' prefix or not?
+    return [this.flagged && `flagged-${this.flagged}`];
```

Comment:
No need to wrap in array if you only have 1 class:
```suggestion
    // TODO: Should we use the 'flagged-' prefix or not?
    return this.flagged;
```